### PR TITLE
[Pages framwork guides]: Add Deploy a Hexo site

### PR DESCRIPTION
**Pull request by DejavuMoe**
- Pages Framework guides: Add Deploy a Hexo site
- Deploy a Blazor site:Title case remains consistent with other items

### CODE REVIEW

**Comment by kristianfreeman** [comment]
@deadlypants1973 to do PCX review.

someone from dev advocacy to verify instructions and walk through

File: content/pages/framework-guides/deploy-a-hexo-site.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+```sh
+$ brew install hexo
+```
+
+{{<render file="_tutorials-before-you-start.md">}}
+
+## Creating a new project
+
+With Hexo CLI installed, create a new project by running the `hexo new` command in your terminal:
```
Comment (lauragift21) [inline content/pages/framework-guides/deploy-a-hexo-site.md]: Did you mean the `hexo init` command? 

```suggestion
With Hexo CLI installed, create a new project by running the `hexo init` command in your terminal:
```

Comment (DejavuMoe) [reply content/pages/framework-guides/deploy-a-hexo-site.md]: Yes, 🥲 I got the commands mixed up incorrectly…

File: content/pages/framework-guides/deploy-a-hexo-site.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Creating a post
+
+Create a new post to give your Hexo site some initial content. Run the `hexo new` command in your terminal to generate a new post:
+
+```sh
+$ hexo new "hello hexo"
+```
+
+Inside of `hello-hexo.md`, use Markdown to write the content of the article, you can customize the tags, categories or other variables in the article, refer to the [Front Matter section](https://hexo.io/docs/front-matter) of the [Hexo documentation](https://hexo.io/docs/)
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-hexo-site.md]:
```suggestion
Inside of `hello-hexo.md`, use Markdown to write the content of the article. You can customize the tags, categories or other variables in the article. Refer to the [Front Matter section](https://hexo.io/docs/front-matter) of the [Hexo documentation](https://hexo.io/docs/) for more information.
```

File: content/pages/framework-guides/deploy-a-hexo-site.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+After deploying your site, you will receive a unique subdomain for your project on `*.pages.dev`.
+Every time you commit new code to your Hexo site, Cloudflare Pages will automatically rebuild your project and deploy it. You will also get access to [preview deployments](/pages/platform/preview-deployments/) on new pull requests, so you can preview how changes look to your site before deploying them to production.
+
+## Using a specific Node.js version
+
+Some Hexo themes or plugins have additional requirements for Node.js version, you can set the environment variable `NODE_VERSION` under Settings to use a specific Node.js version for Hexo.
+
+For example, `NODE_VERSION`: `14.17.5`
+
+## Learn more
```
Comment (deadlypants1973) [inline content/pages/framework-guides/deploy-a-hexo-site.md]:
```suggestion
## Related resources
```

**Review by deadlypants1973 — COMMENTED** [review]
Style guide review before merge

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**